QueryGenie/GenericQueryable.swift

```diff
@@ -1,160 +0,0 @@
-//
-//  GenericQueryable.swift
-//
-//  Created by Anthony Miller on 12/28/16.
-//
-
-import Foundation
-
-/// A query of items of a generic type `Element`.
-public protocol GenericQueryable: Queryable {
```

Comment:
I like this, since `Queryable` is a `Sequence` we can just use the `Element` type of the sequence. It's already inherently "generic".

Reply:
Doh, that totally makes sense. Good catch.